idc_index/index.py

```diff
@@ -665,7 +665,7 @@ def _validate_update_manifest_and_get_download_size(
             SELECT
                 manifest_cp_cmd,
                 REGEXP_EXTRACT(manifest_cp_cmd, '(?:.*?\\/){{3}}([^\\/?#]+)', 1) AS manifest_crdc_series_uuid,
-                REGEXP_REPLACE(regexp_replace(manifest_cp_cmd, 'cp ', ''), '\\s[^\\s]*$', '') AS s3_url,
+                TRIM(REGEXP_REPLACE(regexp_replace(manifest_cp_cmd, 'cp ', ''), '\\s[^\\s]*$', '')) AS s3_url,
```

Comment:
I think this will be easier to read and maintain, and more robust. What do you think?

```suggestion
                REGEXP_EXTRACT(manifest_cp_cmd, 's3://\S+') AS s3_url,
```